### PR TITLE
Add design proposal with splash image and dark background

### PR DESCRIPTION
**Pull request by ssaavedra**
This is a design proposal for the 2020 event page designed by @sofipros and implemented by me.

I did not resize the background image in case we want the hi-res one; not sure if Netlify optimizes this directly. If it doesn't we can also commit a lower-res/lower-quality image for faster load times, as part of this or a separate PR if the design proposal is accepted.

### CODE REVIEW

**Review by dpaez — APPROVED** [review]

**Comment by ssaavedra** [comment]
In the last commit (also in the commitlog):

- Add responsive tag for viewport size on mobile devices
- Add shadows to the buttons
- Fix button:hover bg to match the background
- Fix splash background to be always centered

**Review by martinheidegger — APPROVED** [review]
I like that its an incremental improvement. Good progress! Thanks @ssaavedra & @sofipros

File: website/content/2020/index.md [hunk 1]
```diff
@@ -1,15 +1,16 @@
 ---
 title: Dat Conference 2020
 layout: layout-2020
+header_title: <h1>Dat Conference 2020</h1>
```
Comment (dpaez) [inline website/content/2020/index.md:4]:
```suggestion
header_title: <h1>Dat Community Ecosystem Conference 2020</h1>
```

**Review by dpaez — APPROVED** [review]

File: website/content/2020/index.md (hunk 1 above)
Comment (martinheidegger) [reply website/content/2020/index.md:4]: @dpaez let's put that in a follow up!

**Pull request merged**